zerver/actions/message_send.py

```diff
@@ -210,11 +214,25 @@ def get_recipient_info(
         # of this function for different message types.
         assert stream_topic is not None
 
+        possibly_topic_wildcard_mentioned_user_ids: AbstractSet[int] = set()
```

Comment:
`topic_participant_user_ids` sounds better. Changed.

---

> The uncertainty here is not regarding the user IDs set which we want, but rather regarding if there was an actual topic mention in the message, correct?

I was looking at it from a different perspective: since we are uncertain about @topic mention, we are uncertain about the user_ids contained in the list to be notified later in the code flow. (Thinking this way is confusing, actually; I agree.)